Formula/gemmi.rb

```diff
@@ -0,0 +1,20 @@
+class Gemmi < Formula
+  desc "Macromolecular crystallography library and utilities"
+  homepage "https://project-gemmi.github.io/"
+  url "https://github.com/project-gemmi/gemmi/archive/refs/tags/v0.4.9.tar.gz"
+  sha256 "1e5a4e4351e5860d70c3aa4ebc762b3f098160dad936e1775bd1d165575516bf"
+  license "MPL-2.0"
+  head "https://github.com/project-gemmi/gemmi.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/gemmi", "--help"
```

Comment:
You can use `assert_match` to ensure that `gemmi --help` has run correctly (didn't fail). You'll see examples of using `assert_match` and `shell_output` in other formulae.

Reply:
I'm now using assert_matches. Thanks for the pointer.